src/main/java/cz/inventi/kpj/spring/springintroduction/impl/GreetingFileRepositoryImpl.java

```diff
@@ -0,0 +1,19 @@
+package cz.inventi.kpj.spring.springintroduction.impl;
+import cz.inventi.kpj.spring.springintroduction.GreetingConfigurationProperties;
+import cz.inventi.kpj.spring.springintroduction.persistence.GreetingRepository;
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@Data
```

Comment:
In general, I don't use @DaTa annotation on the service. I don't want to make dependencies visible outside the class. (You don't want to allow call GreetingFileRepositoryImpl.getGreetingConfigurationProperties();)